lib/factories/ppt/slides.js

```diff
@@ -229,12 +250,16 @@ class SlideFactory {
         let slideKey = `ppt/slides/${slide.name}.xml`;
         let objectId = slide.getNextObjectId();
 
-        let shapeX = options.x ? PptxUnitHelper.fromPixels(options.x) : 0;
-        let shapeY = options.y ? PptxUnitHelper.fromPixels(options.y) : 0;
-        let shapeCx = options.cx ? PptxUnitHelper.fromPixels(options.cx) : 2819400;
-        let shapeCy = options.cy ? PptxUnitHelper.fromPixels(options.cy) : 381000;
+        let shapeX = options.x !== undefined ? PptxUnitHelper.fromPixels(options.x) : 0;
```

Comment:
same comment about the default variable assignment above.

Reply:
Done.